src/azure-cli-core/azure/cli/core/style.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Support styled output.
+
+Currently, only color is supported, underline/bold/italic may be supported in the future.
+
+Design spec:
+https://devdivdesignguide.azurewebsites.net/command-line-interface/color-guidelines-for-command-line-interface/
+
+For a complete demo, see `src/azure-cli/azure/cli/command_modules/util/custom.py` and run `az demo style`.
+"""
+
+import sys
+from enum import Enum
+
+from colorama import Fore
+
+
+class Style(str, Enum):
+    PRIMARY = "primary"
+    SECONDARY = "secondary"
+    IMPORTANT = "important"
+    ACTION = "action"  # name TBD
```

Comment:
@chenlomis, what style name should we use for light blue?

Reply:
Hi there
Am slightly confused
Did you mean finding an internal short name for the the bright blue?
https://devdivdesignguide.azurewebsites.net/command-line-interface/color-guidelines-for-command-line-interface/#action-colors